app/services/induction/set_cohort_induction_programme.rb

```diff
@@ -55,4 +63,10 @@ def programme_attrs
 
     attrs
   end
+
+  def set_appropriate_body
+    Induction::SetSchoolCohortAppropriateBody.call(school_cohort:,
+                                                   appropriate_body_id: appropriate_body,
+                                                   appropriate_body_appointed:)
```

Comment:
Do you think it might make sense to remove this call from here and invoke it from the caller?  Would keep the concerns a bit more focussed and clean up the number of params being passed in?
Fine to leave it if you think we'll always need to set the AB info when we are setting the programme choice